worlds/jakanddaxter/docs/en_Jak and Daxter The Precursor Legacy.md

```diff
@@ -0,0 +1,197 @@
+prerequisite moves that you must also have in order to use them (e.g. Crouch Jump is dependent on Crouch). Jak will only
+be able to run, swim (including underwater), perform single jumps, and shoot yellow eco from his goggles ("firing from
+the hip" requires Punch). Note that Flut Flut and the Zoomer will have access to their full movement sets at all times.
+
+You can turn off Move Rando at any time in the game by opening the game's menu, navigate to `Options`, 
+then `Archipelago Options`, then `Move Randomizer`. This will give you access to the full movement set again.
+
+## What are the movement options in Move Randomizer?
+| Move Name       | Prerequisite Moves |
+|-----------------|--------------------|
+| Crouch          |                    |
+| Crouch Jump     | Crouch             |
+| Crouch Uppercut | Crouch             |
+| Roll            |                    |
+| Roll Jump       | Roll               |
+| Double Jump     |                    |
+| Jump Dive       |                    |
+| Jump Kick       |                    |
+| Punch           |                    |
+| Punch Uppercut  | Punch              |
+| Kick            |                    |
+
+## How do I know which moves I have?
+Open the game's menu, navigate to `Options`, then `Archipelago Options`, then `Move Tracker`.
+This will show you a list of all the moves in the game. 
+- Gray items indicate you do not possess that move.
+- Yellow items indicate you possess that move, but you are missing its prerequisites.
+- Light blue items indicate you possess that move, as well as its prerequisites.
+
+## What does Orbsanity do?
+If you enable Orbsanity, Precursor Orbs will be turned into ordered lists of progressive checks. Every time you collect 
+a "bundle" of the correct number of orbs, you will trigger the next release in the list. Likewise, these bundles of orbs 
+will be added to the item pool to be randomized. There are several options to change the difficulty of this challenge. 
+
+- "Per Level" Orbsanity means the lists of orb checks are generated and populated for each level in the game.
+    - (Geyser Rock, Sandover Village, etc.)
+- "Global" Orbsanity means there is only one list of checks for the entire game.
+    - It does not matter where you pick up the orbs, they all count toward the same list.
+- The options with "Bundle Size" in the name indicate how many orbs are in a "bundle." This adds a number of Items 
+  and Locations to the pool inversely proportional to the size of the bundle.
+    - For example, if your bundle size is 20 orbs, you will add 100 items to the pool. If your bundle size is 250 orbs,
+      you will add 8 items to the pool.
+
+### A WARNING ABOUT ORBSANITY OPTIONS
+
+Unlike other settings, you CANNOT alter Orbsanity options after you generate a seed and start a game. **If you turn 
+Orbsanity OFF in the middle of an Orbsanity game, you will have NO way of completing the orb checks.** This may cause
```

Comment:
Agreed, and there is a way to lock down settings in the OpenGOAL pause menu, but I just need to make sure their mechanism for locking still allows us to change the setting by sending the right command to the game when we connect to the AP server. Research item.

Reply:
Just tested this and yes, you can lock down the settings _and_ still change them with the correct command. The locking mechanism is actually a lambda function, so you can tell it lock on a given condition. I chose to ask whether or not the game has the debug flag set to True, because we disable that flag when you connect to the AP server, but it also allows for easy testing and troubleshooting of the mod as a standalone program. Quite happy with this change, thanks for the feedback!

Reply:
![image](https://github.com/user-attachments/assets/bb5ba541-99b7-4552-9073-f51b6fe1ff3d)
![image](https://github.com/user-attachments/assets/867cfc9d-f1f3-4ece-af3b-90c9eec3ab3d)